app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -390,7 +386,9 @@ def hide_button?(id)
                      id !~ /^history_\d*/ &&
                      !id.starts_with?("dialog_") && !id.starts_with?("miq_task_") &&
```

Comment:
@romanblanco without this condition, the `#hide_button?` method would return true for all the dialog buttons thus skipping the call of their `#visible?` method.